anchor_gpt/prompt.py

```diff
@@ -2,30 +2,35 @@
 import numpy as np
 
 class Prompt:
-    def __init__(self, text, scores=[], id=None, embeddings=None):
+    def __init__(self, text, response, scores={}, id=None, embeddings=None):
         self.text = text
+        self.response = response
         self.id = id or uuid.uuid4()
         self.embeddings = embeddings
         self.scores = scores
-    
+
     def _set_store(self, store):
         self.store = store
-    
+
     def add_score(self, score):
-        '''
+        """
         Add a score to the prompt.
-        score: any. Will be passed in the prompts to the PromptLogger retriever function to allow custom scoring.
-        '''
-        self.scores.append(score)
+
+        Parameters:
+            score: any. Will be passed in the prompts to the PromptLogger retriever function to allow custom scoring.
```

Comment:
this isn't `any` anymore since it's called with `extend` - I'd rename this `update_score`

Reply:
ok